nce/NCE_MP_Pytorch/README.md

```diff
@@ -0,0 +1,110 @@
+# MP-CNN PyTorch Implementation
```

Comment:
This basically copies what I had for MP-CNN, but this is NCE on top of MP-CNN. You should update everything in this README and indicate clearly if some measurement is MP-CNN only or MP-CNN with NCE.

Reply:
Please address this comment and do not copy the MP-CNN README. I would add a new row for each dataset with NCE in the mp_cnn README.

Reply:
Or you can make a link to the MP-CNN README and do not repeat instructions that are already there. In this README just have the tables showing performance with and without NCE.